### ISSUE TITLE
Update Documentation

### ISSUE THREAD

**Issue by sanzoghenzo** [mamba-org/quetz#596]
Hi there! Here I come, 2 years after my first deployment of quetz.

TL;DR: Add all the authentication plugins to the authentication doc page and explain the use of the environment variables in the configuration page.

Long Version: my company was acquired by another one, and we are moving from google workspace to microsoft 365.
I need to switch quetz authentication, and I headed to the documentation to see if there were some updates in these two years.
Having found nothing, I updated my fork to start implementing a micrisoft oidc plugin, and then I found that the azuread plugin was already there.

Also, looking at the configuration page, I see that there a few environment variables declared, but searching through the issues and looking at the code it's clear that every config done in the toml file is now possible via environment variables.
This has to be clarified in the documentation!

**Comment by wolfv** [comment]
Thanks for the note. Indeed, we do support azure login. Do you have time to update the docs?

**Comment by sanzoghenzo** [comment]
I could give it a try, but honestly I don't know what's changed in these 2 years, so I'm the last qualified person to do that.

I opened this issue exactly because of that, and because I feel that new user would hugely benefit of updated documentation!
